src/workerd/api/wpt/urlpattern-test.js

```diff
@@ -0,0 +1,175 @@
+// Copyright (c) 2017-2022 Cloudflare, Inc.
+// Licensed under the Apache 2.0 license found in the LICENSE file or at:
+//     https://opensource.org/licenses/Apache-2.0
+
+import { run } from 'harness';
+
+export const urlpatternCompare = run('urlpattern-compare-tests.js', {
+  expectedFailures: [
```

Comment:
```suggestion
  expectedFailures: [
    // Each of these *ought* to pass. They are included here because we
    // know they currently do not. Each needs to be investigated.
```

Reply:
Yep